models/olids/stable/stable_ndoo_hashed.sql

```
@@ -0,0 +1,36 @@
+    lds_batch_id,
+    lds_file_id,
+    lds_dataset_id,
+    effective_from,
+    effective_to,
+    is_latest,
+    lakehouse_date_processed,
+    high_watermark_date_time
+from {{ ref('base_olids_ndoo_hashed') }}
+
+{% if is_incremental() %}
+    where lds_start_date_time > (select max(lds_start_date_time) from {{ this }})
```

SQLFluff (4.1.0)

[error] Line is too long (81 > 80).

(LT05)